Application/src/model/Parser.java

```diff
@@ -62,20 +65,25 @@ public void parseClassDiagram(String inputJSON) {
             StringBuilder string = new StringBuilder();
 
             string.append("{");
+            string.append("class_diagram,");
+            string.append(DiagramView.getDiagramViewInView().getTab().getId());
+            string.append(",");
             string.append(UniqueCounter.getString());
             string.append(",[");
 
             // Add the classes.
             for (Classes s: cd.classes) {
-                string.append("[");
+                string.append("['");
```

Comment:
Why did you change this?

Reply:
Because an idiot requirested atoms to use ' ' ' ' ' ' ' '

Reply:
Who is idiot? specify

Reply:
@pontuslaestadius how about String.toLowerCase();